src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+/**
+ * This represents the Duke Chat bot.
+ */
 public class Duke {
+
+    private static boolean done = false;
+    private static Task[] list = new Task[100];
+    private static int listIndexTracker = 0;
+    private static boolean error = false;
+
+    private final static String LINES = "    ____________________________________________________________";
+    private final static String TAB = "    ";
+    private final static String GREETING = "    Hello... I'm Sadge Duke\n    What can I do for you? :(";
+    private final static String GOODBYE = "    Bye. I'll miss you...";
+    private final static String TODO = "todo";
+    private final static String DEADLINE = "deadline";
+    private final static String EVENT = "event";
+    private final static int DESCRIPTION = 0;
+    private final static int DATETIME = 1;
+
+
+    /**
+     * Main function that is called upon program execution.
+     *
+     * @param args System Arguments added to program.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        /**
+         * ASCII art source: https://www.twitchquotes.com/copypastas/2059
+         */
+        String sadge = "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⠿⠛⠋⠉⣉⣉⠙⠿⠋⣠⢴⣊⣙⢿⣿⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⡿⠟⠋⠁⠀⢀⠔⡩⠔⠒⠛⠧⣾⠊⢁⣀⣀⣀⡙⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⠟⠛⠁⠀⠀⠀⠀⠀⡡⠊⠀⠀⣀⣠⣤⣌⣾⣿⠏⠀⡈⢿⡜\n"
+                + "   ⣿⣿⣿⠿⠛⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠡⣤⣶⠏⢁⠈⢻⡏⠙⠛⠀⣀⣁⣤⢢\n"
+                + "   ⣿⠋⠁⠀⠀⠀⠀⠀⠀⠀⠀⠀⠰⣄⡀⠣⣌⡙⠀⣘⡁⠜⠈⠑⢮⡭⠴⠚⠉⠀\n"
+                + "   ⠁⠀⢀⠔⠁⣀⣤⣤⣤⣤⣤⣄⣀⠀⠉⠉⠉⠉⠉⠁⠀⠀⠀⠀⠀⠁⠀⢀⣠⢠\n"
+                + "   ⡀⠀⢸⠀⢼⣿⣿⣶⣭⣭⣭⣟⣛⣛⡿⠷⠶⠶⢶⣶⣤⣤⣤⣶⣶⣾⡿⠿⣫⣾\n"
+                + "   ⠇⠀⠀⠀⠈⠉⠉⠉⠉⠉⠙⠛⠛⠻⠿⠿⠿⠷⣶⣶⣶⣶⣶⣶⣶⣶⡾⢗⣿⣿\n"
+                + "   ⣦⣄⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣠⣴⣿⣶⣾⣿⣿⣿\n"
+                + "   ⣿⣿⣿⣷⣶⣤⣄⣀⣀⣀⡀⠀⠀⠀⠀⠀⠀⢀⣀⣤⣝⡻⣿⣿⣿⣿⣿⣿⣿⣿\n"
+                + "   ⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣦⡹⣿⣿⣿⣿⣿⣿\n";
+
+        /**
+         * ASCII art source: https://fsymbols.com/text-art/
+         */
+        String text = "█▀ ▄▀█ █▀▄ █▀▀ █▀▀   █▀▄ █░█ █▄▀ █▀▀\n"
+                + "▄█ █▀█ █▄▀ █▄█ ██▄   █▄▀ █▄█ █░█ ██▄";
+
+        System.out.println("Hello from\n" + sadge + text);
+
+        System.out.println(LINES);
+        System.out.println(GREETING);
+        System.out.println(LINES);
```

Comment:
Perhaps you can extract these lines of code to a new method like `printWelcomeBanner()`?